src/lib.rs

```diff
@@ -130,6 +132,14 @@ impl Fairing for RocketSentry {
 
     async fn on_ignite(&self, rocket: Rocket<Build>) -> fairing::Result {
         let figment = rocket.figment();
+        let profile_name = figment.profile().to_string();
+
+        // Set Sentry's environment based on Rocket profile
+        let environment: &'static str = match profile_name {
+            debug if debug == "debug" => "development",
+            release if release == "release" => "production",
```

Comment:
I find this `release if release == "release"` really had to parse. How about...

```rust
        let environment = match profile_name.as_str() {
            "debug" => Cow::Borrowed("development"),
            "release" => Cow::Borrowed("production"),
            _ => Cow::Owned(profile_name),
        };
```

Or actually, I think if you forego the `to_string()` and and compare values with `if()`, you can avoid a string copy entirely in those cases.

Reply:
I wasn't happy with the result either. Using `Cow` improve it nicely.   
I'm afraid the `to_string()`  is necessary, as a profile is a [UncasedStr](https://docs.rs/figment/0.10.19/figment/value/struct.UncasedStr.html) which I don't see how to use in the match.